.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "crates/utils/translations"]
+	path = crates/utils/translations
+	url = https://github.com/LemmyNet/lemmy-translations.git
```

Comment:
Not exactly sure why the build is failing, but this is probably the issue. Looks like this on joinlemmy:
```
[submodule "lemmy-translations"]                                                     
path = lemmy-translations
url = https://github.com/lemmynet/lemmy-translations
branch = main
```

Reply:
Fixed, passes now